bigquery_etl/dryrun.py

```diff
@@ -321,14 +61,41 @@ def __init__(
         self.use_cloud_function = use_cloud_function
         self.client = client if use_cloud_function or client else bigquery.Client()
         self.respect_skip = respect_skip
+        self.dry_run_url = ConfigLoader.get("dry_run", "function")
         try:
             self.metadata = Metadata.of_query_file(self.sqlfile)
         except FileNotFoundError:
             self.metadata = None
 
+    @staticmethod
+    def skipped_files() -> Set[str]:
+        """Return files skipped by dry run."""
+        skip_files = {
+            file
+            for skip in ConfigLoader.get("dry_run", "skip", fallback=[])
+            for file in glob.glob(
+                skip,
+                recursive=True,
+            )
+        }
+
+        # update skip list to include renamed queries in stage.
+        test_project = ConfigLoader.get("dry_run", "test_project", fallback="")
+        skip_files.update(
+            [
+                p
+                for f in [Path(s) for s in skip_files]
+                for p in glob.glob(
+                    f"sql/{test_project}/{f.parent.parent.name}*/{f.parent.name}/{f.name}",
+                    recursive=True,
+                )
+            ]
+        )
+        return skip_files
+
```

Comment:
Load skipped files from config

Reply:
This seems like it will still have the problem the current process has due to `--remove-updated-artifacts` being specified for the staging deploy where the original glob patterns won't match all the appropriate the files because they've already been moved/removed.

Reply:
Updated to replace existing project with the test project before resolving the glob patterns